### PR TITLE
move staging-central to staging-west

### PR DESCRIPTION
**Pull request by Liujingfang1**
- Since the staging cluster created is in us-west. So I moved the environment to staging-west.
- Updated the namespace to `go-app`, which is managed by the root repo.
- Fixed a bug in the deployment patch file.

### CODE REVIEW

**Comment by Liujingfang1** [comment]
cc @janetkuo @yuwenma @nan-yu @haiyanmeng

File: starter-repos/app-template/config/envs/prod-central/kustomization.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # Customize the namespace to be the namespace name used for prod-central environment.
 # This change should be made before `appcel env add prod-central`.
 # Once the prod-central environment is added, the namespace shouldn't be changed.
-namespace: prod-central
+namespace: go-app
```
Comment (janetkuo) [inline starter-repos/app-template/config/envs/prod-central/kustomization.yaml:18]: So the same `go-app` namespace will be used to deploy the app in staging/prod clusters?

Comment (Liujingfang1) [reply starter-repos/app-template/config/envs/prod-central/kustomization.yaml:18]: Yes, that is the namespace I put in the root repo.

**Comment by nan-yu** [comment]
Can you also make the following changes in this PR?
- https://github.com/GoogleCloudPlatform/solutions-modern-cicd-anthos/blob/app-delivery/starter-repos/app-template/appctl/prepare.yaml#L48
- https://github.com/GoogleCloudPlatform/solutions-modern-cicd-anthos/blob/app-delivery/starter-repos/app-template/appctl/validate.yaml#L46
- https://github.com/GoogleCloudPlatform/solutions-modern-cicd-anthos/blob/app-delivery/starter-repos/app-template/skaffold/render.yaml#L33
- https://github.com/GoogleCloudPlatform/solutions-modern-cicd-anthos/blob/app-delivery/starter-repos/app-template/skaffold/render.yaml#L36

Thanks.

**Comment by Liujingfang1** [comment]
@nan-yu I searched and replaced all the `staging-central`.

**Review by nan-yu — APPROVED** [review]
/lgtm

**Review by janetkuo — APPROVED** [review]

**Pull request merged**